Xamarin.RevenueCat.iOS/ApiDefinitions.cs

```diff
@@ -5,1044 +5,1613 @@
+		// @property (readonly, nonatomic) NSUInteger hash;
+		[Export("hash")] nuint Hash { get; }
+
+		// @property (readonly, copy, nonatomic) NSString * _Nonnull description;
+		[Export("description")] string Description { get; }
+
+		// @property (readonly, nonatomic, strong) SKPaymentTransaction * _Nullable sk1Transaction;
+		[NullAllowed, Export("sk1Transaction", ArgumentSemantic.Strong)]
+		SKPaymentTransaction Sk1Transaction { get; }
+	}
+
+// @interface RCStorefront : NSObject
+	[BaseType(typeof(NSObject))]
+	[DisableDefaultCtor]
+	interface RCStorefront
+	{
+		// @property (readonly, copy, nonatomic) NSString * _Nonnull countryCode;
+		[Export("countryCode")] string CountryCode { get; }
+
+		// @property (readonly, copy, nonatomic) NSString * _Nonnull identifier;
+		[Export("identifier")] string Identifier { get; }
+
+		// -(BOOL)isEqual:(id _Nullable)object __attribute__((warn_unused_result("")));
+		[Export("isEqual:")]
+		bool IsEqual([NullAllowed] NSObject @object);
+
+		// @property (readonly, nonatomic) NSUInteger hash;
+		[Export("hash")] nuint Hash { get; }
+
+		// @property (readonly, copy, nonatomic) NSString * _Nonnull description;
+		[Export("description")] string Description { get; }
+
+		// @property (readonly, nonatomic, strong) SWIFT_AVAILABILITY(maccatalyst,introduced=13.1) SKStorefront * sk1Storefront __attribute__((availability(maccatalyst, introduced=13.1))) __attribute__((availability(watchos, introduced=6.2))) __attribute__((availability(tvos, introduced=13.0))) __attribute__((availability(macos, introduced=10.15))) __attribute__((availability(ios, introduced=13.0)));
+		[Watch(6, 2), TV(13, 0), MacCatalyst(13, 1), Mac(10, 15), iOS(13, 0)]
+		[Export("sk1Storefront", ArgumentSemantic.Strong)]
+		SKStorefront Sk1Storefront { get; }
+
+		// @property (readonly, nonatomic, strong, class) RCStorefront * _Nullable sk1CurrentStorefront __attribute__((availability(maccatalyst, introduced=13.1))) __attribute__((availability(watchos, introduced=6.2))) __attribute__((availability(tvos, introduced=13.0))) __attribute__((availability(macos, introduced=10.15))) __attribute__((availability(ios, introduced=13.0)));
+		[Watch(6, 2), TV(13, 0), MacCatalyst(13, 1), Mac(10, 15), iOS(13, 0)]
+		[Static]
+		[NullAllowed, Export("sk1CurrentStorefront", ArgumentSemantic.Strong)]
+		RCStorefront Sk1CurrentStorefront { get; }
+	}
+
+// @interface RCSubscriptionPeriod : NSObject
+	[BaseType(typeof(NSObject))]
+	[DisableDefaultCtor]
+	interface RCSubscriptionPeriod
+	{
+		// @property (readonly, nonatomic) NSInteger value;
+		[Export("value")] nint Value { get; }
+
+		// @property (readonly, nonatomic) enum RCSubscriptionPeriodUnit unit;
+		[Export("unit")] RCSubscriptionPeriodUnit Unit { get; }
+
+		// -(BOOL)isEqual:(id _Nullable)object __attribute__((warn_unused_result("")));
+		[Export("isEqual:")]
+		bool IsEqual([NullAllowed] NSObject @object);
+
+		// @property (readonly, nonatomic) NSUInteger hash;
+		[Export("hash")] nuint Hash { get; }
```

Comment:
imho the override declaration is needed. otherwise the member would be hidden.

Reply:
I don't think we can override hash. Like from before we don't override hash https://github.com/thisisthekap/Xamarin.RevenueCat.iOS/blob/master/Xamarin.RevenueCat.iOS/ApiDefinitions.cs#L841

Reply:
Ah, got it. This is needed only if we get a compiler warning that a parent class member declaration would be hidden.